src/Akka.Streams.Kafka.Tests/Integration/RebalanceIntegrationTests.cs

```diff
@@ -87,7 +88,11 @@ public async Task ShouldReBalanceWithoutArgumentExceptions()
         var killSwitch3 = CreateKillableStream(topic, settings, probe1.Ref, "consumer3");
 
         // make sure all 10 messages got processed
-        var msgs1 = await probe1.ReceiveNAsync(10).Cast<ConsumeResult<Null, string>>().ToListAsync();
+        await foreach (var msg in probe1.ReceiveNAsync(10))
+        {
```

Comment:
Not necessary - the previous code would blow up anyway if a `StreamFailed` message arrived